stories/accordion.stories.js

```diff
@@ -43,17 +42,31 @@ const doAlert = (msg) => window.alert(msg); // eslint-disable-line no-alert, no-
 const variantOptions = ['default', 'minimal'];
 
 export default {
-  title: 'Accordion'
+  title: 'Accordion',
+  argTypes: {
```

Comment:
Arg types for our existing stories are often inferrable, but we need to specify them explicitly for `select`s and their options and a few `date`s.  It can also be handy to specify the `name` when it either can't be inferred or needs to be customized.